Resources/Storage/StorageElement.py

```diff
@@ -370,7 +370,7 @@ def getOccupancy(self, unit='MB', **kwargs):
         It loops over the different Storage Plugins to query it.
 
         :params occupancyLFN: (named param) LFN where to find the space reporting json file on the storage
-                              The json file should contain the Free and Total space in MB.
+                              The json file should contain the Free and Total space in B.
```

Comment:
I don't remember why in the past we assumed (thought?) that we would have found MB. So, why now Bytes?

Reply:
Because the first test file I ever received was in MB... now it's in Bytes, which btw makes more sense